odbc/src/api/functions.rs

```diff
@@ -103,15 +122,24 @@ macro_rules! panic_safe_exec {
         let result = panic::catch_unwind(function);
         panic::set_hook(previous_hook);
         match result {
-            Ok(sql_return) => return sql_return,
+            Ok(sql_return) => {
+                #[allow(unused_variables)]
+                let trace = trace_call_and_outcome(function_name!(), &sql_return);
```

Comment:
[nit] Could you create the variable before the match since you use it in both arms of the match?

Reply:
The `trace_call_and_outcome()` function uses a `sql_return` variable that is only in the match branch so it seems valid to have this in both arms of the branch.